src/subcommand/decode.rs

```diff
@@ -0,0 +1,28 @@
+use super::*;
+
+#[derive(Serialize, Eq, PartialEq, Deserialize, Debug)]
+pub struct Output {
+  pub inscriptions: Vec<Inscription>,
+}
+
+#[derive(Debug, Parser)]
+pub(crate) struct Decode {
+  transaction: PathBuf,
```

Comment:
what would you think about making a flag to read from stdin? would be useful for doing things like running mempool transactions through it

Reply:
Done!